rts/Sim/Units/CommandAI/BuilderCAI.cpp

```diff
@@ -137,7 +137,7 @@ CBuilderCAI::CBuilderCAI(CUnit* owner):
 
 		c.action    = "reclaim";
 		c.name      = "Reclaim";
-		c.tooltip   = c.name + ": Sucks in the metal/energy content of a unit/feature and add it to your storage";
+		c.tooltip   = c.name + ": Sucks in the metal/energy content of a unit/feature\nand add it to your storage";
```

Comment:
also add=>adds, I'll do it myself, just commenting to remind me as I'm going to sleep